addons/website_event_booth/static/src/js/booth_register.js

```diff
@@ -131,6 +190,48 @@ publicWidget.registry.boothRegistration = publicWidget.Widget.extend({
         }
     },
 
+    /**
+     * Submit the confirmation form if no errors are present after validation.
+     *
+     * If the submission succeed, we replace the form with a success message template.
+     *
+     * @param ev
+     * @return {Promise<void>}
+     * @private
+     */
+    async _onConfirmClick(ev) {
```

Comment:
So you have a "onSubmit" and "onConfirm" that do different things?

Could be nice to name them a little more appropriately based on what they submit and what they confirm.
From what I understand one of them is the booth selection and the other is when you confirm your order?